docs/atomic-ps.1.md

```diff
@@ -24,11 +27,21 @@ Using --all will list all the installed containers.
 **-a** **--all**
   Print all the installed containers
 
-**-n** **--noheading**
-  Do not print heading when listing the containers
+**-f** **--filter**
+  Filter output based on given filters in the format of VARIABLE=VALUE
 
```

Comment:
Give me more data here on what is filterable.

+        filterables = ["container", "image", "command", "created", "status", "runtime"]